Modules/Core/Mesh/test/itkCellInterfaceTest.cxx

```diff
@@ -142,91 +142,43 @@ TestCellInterface(std::string name, TCell * aCell)
 int
 itkCellInterfaceTest(int, char *[])
 {
-  int status;
+  int status = EXIT_FAILURE;
```

Comment:
This _status_ seems to be useless, if I see it correctly _TestCellInterface_ always return EXIT_SUCCESS. And _status_ is overwritten by each test. It is not a problem, just does not make much sense.

Reply:
I missed that, sorry. I will amend the commit later today. Thanks.

Reply:
Done in 88858aa.